dist/js/page-specific/citysourced-viewer-googleMaps.js

```diff
@@ -6,6 +6,8 @@ baltimoreCounty.pageSpecific.viewerGoogleMaps = (function (googleMaps, undefined
 		autocomplete,
 		apiKey = 'AIzaSyAqazsw3wPSSxOFVmij32C_LIhBSuyUNi8',
```

Comment:
It will be eventually, but not initially. I discussed with Dan a while back and he's OK with it for now.